Classes/Settings/SettingsController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  TestSettingsController.swift
+//  MobileOrg
+//
+//  Created by Jamie Conlon on 04/01/2017.
+//  Copyright © 2017 Sean Escriva. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class SettingsController: UITableViewController {
+    
+    @IBOutlet weak var appBadgeSwitch: UISwitch!
```

Comment:
The existing Swift files have a 2 spaces indentation. Would be nice if we could keep this consistent.